File: microverse/course/projects/05 javascript/testing-practice-js-jest/js/functions/calculator.js
const calculator = (() => {
  const add = (a, b) => { return a + b }
  const subtract = (a, b) => { return a - b }
  const multiply = (a, b) => { return a * b }
  const divide = (a, b) => { return a / b }
  
  return { add, subtract, multiply, divide }
})()

module.exports = calculator;

File: .git-rewrite/t/theodinproject/ruby programming/calculator-spec-test/lib/calculator.rb
#lib/calculator.rb

class Calculator

  def add(*args)
    args.each.reduce(:+)
  end

  def multiply(*args)
    args.each.reduce(:*)
  end

  def subtract(*args)
    args.each.reduce(:-)
  end

  def divide(*args)
    args.each.reduce(:/)
  end

end
